Modules/xSharePoint/Modules/xSharePoint.Util/xSharePoint.Util.psm1

```diff
@@ -32,6 +32,18 @@ function Get-xSharePointContentService() {
     return [Microsoft.SharePoint.Administration.SPWebService]::ContentService
 }
 
+
+function Get-xSharePointUserProfileSubTypeManager {
+    [CmdletBinding()]
+    param
+    (
+        $context
```

Comment:
My main concern here is that we are internally consistent, at least in a module.  It makes it easier to read and review if we all use the same naming and casing standards.